src/executable-code/executable-fragment.js

```diff
@@ -181,6 +188,57 @@ export default class ExecutableFragment extends ExecutableCodeTemplate {
     }
   }
 
+  removeAllOutputNodes() {
+    const outputNode = this.element.getElementsByClassName("code-output").item(0)
+    while (outputNode && outputNode.lastChild) {
+      outputNode.removeChild(outputNode.lastChild)
+    }
+  }
+
+  applyStateUpdate(stateUpdate) {
+    if (stateUpdate.errors === null) {
+      stateUpdate.errors = []
+    } else if (stateUpdate.errors !== undefined) {
+      this.state.errors.push(...stateUpdate.errors)
+      stateUpdate.errors = this.state.errors
+    }
+
+    Object.keys(stateUpdate).forEach(key => {
+      if (stateUpdate[key] === undefined) {
+        delete stateUpdate[key];
+      }
+    })
+    Object.assign(this.state, stateUpdate, {
+      isShouldBeFolded: this.isShouldBeFolded && stateUpdate.isFoldedButton
+    })
+  }
+
+  renderNewOutputNodes(stateUpdate) {
```

Comment:
Pls, add new `callback` for track output update (looks #83)

Reply:
I added `onOutputAddedToDom` callback.